azure/README.md

```diff
@@ -0,0 +1,38 @@
+# azure
+
+This directory contains the terraform code to create an Azure Service Principal and the Federated Identity credentials which will be used in later Terraform Stacks operations.
+
+## Setup
+
+To execute this configuration you will need to give Terraform access to the relevant Azure account. Refer the [following](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#authenticating-to-azure) terraform provider docs to authenticate with Azure.
+
+Once the Terraform configuration has been applied you should save the following from your Azure account:
+
+1. `Tenant ID` of the Azure AD/Microsoft Entra ID service.
+2. `Application (client) ID` of the registered application created by this terraform setup.
```

Comment:
Could we add this as an output to the configuration, just so it prints it out?

Reply:
Good idea. I will make this change.

Reply:
Added the outputs. Here is an example of how it looks
```bash
Outputs:

client_id = "c22094b1-047d-4221-bc61-82907b50f338"
subscription_id = "1845c88e-c6c2-4dfb-94e1-552aeecb228e"
tenant_id = "8e688f32-c820-4e21-97c7-49b5adee5220"
```